src/io/structure/assemble.rs

```diff
@@ -41,6 +41,7 @@ pub struct Assemble {
     /// between the first layer encountered on either side of the boundary)
     pub vacuum_sep: f64,
     layer_seps: Vec<f64>, // [layer] (length: nlayer - 1)
+    pub atoms: Vec<Element>
```

Comment:
On these internal types, I would call this `elements` for consistency with the rest of rsp2.